webapp/src/lib/components/ButtonBig.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+	export let text: string;
+	export let accent_color: string = '#ff847c';
+</script>
+
+<div class="w-[19rem] h-[7rem] relative flex items-center justify-center">
+	<button class="w-[15rem] h-[4rem] flex justify-center items-center bg-yellow z-20 absolute">
+		<p class="font-contrail text-3xl">{text}</p>
+	</button>
+	<div
+		class="w-[15rem] h-[4rem] right-12 top-8 border-[0.5rem] z-10 -rotate-[4deg] absolute"
```

Comment:
_right-10_ would be closer to the design template and more consistent with the ButtonSmall component.